client/packages/host/src/components/Login/Login.tsx

```diff
@@ -66,6 +66,10 @@ export const Login = () => {
       );
       return `${t('error.account-blocked')} ${formattedTime}`;
     }
+    if (error.detail?.includes('TransactionError')) {
+      return t('error.database-busy');
+    }
```

Comment:
We map it to an unspecified/unexpected internal error. The string is just a helper to indicate what this error was. So we shouldn't really look for a specific string in it. We can say "The server experienced a problem please retry again later" instead.

Reply:
You could display the detail in the tooltip though...

Reply:
I mean could you please check if its an internal error instead of looking into the detail message (which is sqlite specific)

It should be in the `error.message == "InternalError"` field if I am not mistaken

Reply:
It's a database error. The issue you closed is different... I mean I guess it's the same since it's also a database error but the message won't apply...

Reply:
but its both an internal error isn't it: "The server experienced a problem. Please try again later" should still work (even though a migration problem can't fix itself)

Reply:
Makes the message so misleading with migration problem...

Reply:
It still didn't like searching message for UpdateUserError D:

Reply:
A migration problem should hopefully not happen, and then we would need the msg in the tooltip anyway. I think this it s ok to say "The server experienced a problem".

> It still didn't like searching message for UpdateUserError D:

Can you please check for "Internal error" instead? this might need some refactor to keep the original standard error in the error object... Please, let me know if I should have another look at it how to do this.